inception-doc/src/main/resources/META-INF/asciidoc/admin-guide/metrics_monitoring.adoc

```diff
@@ -0,0 +1,89 @@
+// Copyright 2020
+// Ubiquitous Knowledge Processing (UKP) Lab and FG Language Technology
+// Technische Universität Darmstadt
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+= Monitoring the INCEpTION instance
+
+== Available metrics
+
+We expose some metrics of the running INCEpTION instance via JMX. These are currently
+
+* the number of active as well as enabled users
+* the overall number of documents
+* the number of enabled recommenders
+* the number of annotation documents i.e. documents being annotated per user
+
+To make the metrics available `spring.jmx.enabled=true`
+and `monitoring.metrics.enabled=true` 
+must be set in the `settings.properties` file 
+(see <<sect_home_folder>> on this file).
+
+== Setting up metrics exporter
+
+To export the metrics so they can be queried by the monitoring solution https://prometheus.io/[Prometheus],
+you can e.g. use https://github.com/prometheus/jmx_exporter[the JMX exporter] as a java agent.
+
+The JMX exporter can be run as a .jar file that should be placed together with its `config.yml` 
+file next to the INCEpTION .jar file. An example `config.yml` file that exposes metrics from 
```

Comment:
Use the `{product-name}` (I think) placeholder instead of hard-coding "INCEPTION".